internal/cmd/login_test.go

```diff
@@ -148,7 +164,7 @@ var cmpClientHostConfig = cmp.Options{
 		cmpPolymorphicIDNotZero),
 	cmp.FilterPath(
 		opt.PathField(ClientHostConfig{}, "Expires"),
-		cmpApiTimeWithThreshold(5*time.Second)),
+		cmpApiTimeWithThreshold(20*time.Second)),
```

Comment:
Since the expect can take up to 30s I was seeing a few flakes when the config was written more than 5 seconds later. I increased this value to remove the flakes.